site/gatsby-site/cypress/e2e/integration/classificationsEditor.cy.js

```diff
@@ -0,0 +1,443 @@
+      namespace: 'GMF',
+    },
+    {
+      namespace: 'CSETv1',
+    },
+  ].forEach(({ namespace }) => {
+    maybeIt(
+      `Should properly display and store ${namespace} classification values`,
+      { scrollBehavior: 'center' },
+      () => {
+        cy.login(Cypress.env('e2eUsername'), Cypress.env('e2ePassword'));
+
+        cy.conditionalIntercept(
+          '**/graphql',
+          (req) => req.body.operationName == 'FindClassifications',
+          'FindClassifications',
+          {
+            data: {
+              classifications: [],
+            },
+          }
+        );
+
+        cy.visit(incidentURL);
+
+        cy.waitForStableDOM();
+
+        cy.query({
+          query: gql`
+            query TaxasQuery($namespace: String) {
+              taxas(query: { namespace: $namespace }) {
+                namespace
+                field_list {
+                  permitted_values
+                  display_type
+                  short_name
+                  mongo_type
+                }
+              }
+            }
+          `,
+          variables: { namespace },
+        }).then(({ data: { taxas } }) => {
+          for (const taxa of taxas) {
+            const { namespace } = taxa;
+
+            cy.contains('Select a taxonomy').scrollIntoView();
+
+            cy.contains('Select a taxonomy').click();
+
+            cy.contains(namespace).click();
+
+            cy.contains('Add').click();
+
+            cy.waitForStableDOM();
+
+            cy.conditionalIntercept(
+              '**/graphql',
+              (req) =>
+                req.body.operationName == 'UpsertClassification' &&
+                req.body.variables.query.namespace == namespace,
+              `Upsert-${namespace}`,
+              classificationsUpsertMock
+            );
+
+            cy.get(`[data-cy="classifications-editor"] [data-cy="taxonomy-${namespace}"]`).within(
+              () => {
+                cy.contains('Edit').click();
+
+                const selectedValues = {};
+
+                for (const field of taxa.field_list) {
+                  setField({
+                    short_name: field.short_name,
+                    display_type: field.display_type,
+                    permitted_values: field.permitted_values,
+                  }).then((value) => {
+                    selectedValues[field.short_name] = value;
+                  });
+                }
+
+                cy.contains('Submit').click();
+
+                cy.wait(`@Upsert-${namespace}`, { timeout: 20000 }).then((xhr) => {
+                  expect(xhr.request.body.variables.query.namespace).eq(namespace);
+
+                  for (const field of taxa.field_list) {
+                    expect(
+                      xhr.request.body.variables.data.attributes.filter(
+                        (a) => a.short_name == field.short_name
+                      )
+                    ).to.have.lengthOf(1);
+
+                    const skippedFields = [
```

Comment:
I will write tests for these fields in a separate PR, but I want to get this PR out of the way, and this newly added test is already better than what we had previously, which was no tests.